articles/client-platforms/react/10-customizing-lock.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize lock widget.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/10-Customizing-Lock',
+}) %>_
+
+Using Lock widget is great, but eventually you will want to customize the UI. There are several options to do that.
+
+## 1. Lock options
+
+Some UI customization can be done via the `options` parameter when creating a `Lock` instance.
+
+
+### Theme options
+
+You can set custom theme properties like using a different logo, or changing the primary color.
+Just add a `theme` property with custom values.
```

Comment:
I think we could make it clear we're changing AuthService class. maybe with something like: Just add a `theme` option when initializing the auth0 Lock widget... 
Below you see our `AuthService` helper class updated to set a custom theme: